src/libs/ReportUtils.ts

```diff
@@ -1228,7 +1227,8 @@ function isClosedExpenseReportWithNoExpenses(report: OnyxEntry<Report>): boolean
 /**
  * Whether the provided report is an archived room
  */
-function isArchivedRoom(report: OnyxInputOrEntry<Report> | EmptyObject, reportNameValuePairs?: OnyxInputOrEntry<ReportNameValuePairs> | EmptyObject): boolean {
+function isArchivedRoom(reportOrID: OnyxInputOrEntry<Report> | EmptyObject | string, reportNameValuePairs?: OnyxInputOrEntry<ReportNameValuePairs> | EmptyObject): boolean {
+    const report = typeof reportOrID === 'string' ? getReport(reportOrID) : reportOrID;
```

Comment:
Let's access `allReports` directly, we don't need to call `getReport` (it checks the draft report too)